core/src/main/resources/application.yml

```diff
@@ -223,6 +223,7 @@ cb:
         CDP 1.2 - SDX Medium Duty: Apache Hive Metastore, Apache Ranger, Apache Atlas=cdp-sdx-medium-ha-702;
         CDP 1.2 - Streams Messaging Heavy Duty: Apache Kafka, Schema Registry, Streams Messaging Manager=cdp-streaming-702;
         CDP 1.2 - Streams Messaging Light Duty: Apache Kafka, Schema Registry, Streams Messaging Manager=cdp-streaming-small-702;
+        CDP 1.2 - Discovery Data Hub: Apache Solr, Apache Spark, Hue=cdp-ddh-702;
```

Comment:
Not sure it makes sense to release this template back 7.0? This is a very old code line and are we sure that this works?